indexers/src/main/java/org/mousephenotype/cda/indexers/PipelineIndexer.java

```diff
@@ -94,8 +95,9 @@ public RunStatus validateBuild()	throws IndexerException {
 
 
 	private void initialiseSupportingBeans(RunStatus runStatus)
-			throws IndexerException, SQLException {
+			throws IndexerException, SQLException, OWLOntologyCreationException, OWLOntologyStorageException, IOException {
 
```

Comment:
Catch SQLException, OWLOntologyCreationException, OWLOntologyStorageException, IOException and wrap into IndexerException